graftm/run.py

```diff
@@ -44,7 +45,7 @@ def __init__(self, args):
         self.setattributes(self.args)
 
     def setattributes(self, args):
-        
+        self.t = Taxonomy()
```

Comment:
It isn't necessary, I just kept it together with HouseKeeping() and Stats_And_Summary(). Do you think this should not be an instance variable (Does it cause issues?)